create.go

```diff
@@ -46,6 +46,10 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 			Name:  "no-new-keyring",
 			Usage: "do not create a new session keyring for the container.  This will cause the container to inherit the calling processes session key",
 		},
+		cli.IntFlag{
+			Name:  "preserve-file-descriptors",
```

Comment:
Good idea. I think I'll go for `--preserve-fds` to keep it plural. Maybe `--nr-preserve-fds` would be clearer though (but then the length creeps back up...). `--nr-keep-fds`?